java/ql/test/experimental/query-tests/security/CWE-598/SensitiveGetQuery2.java

```diff
@@ -0,0 +1,29 @@
+import java.io.IOException;
+import java.util.Map;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+
+public class SensitiveGetQuery2 extends HttpServlet {
+	// BAD - Tests sending sensitive information in a GET request.
```

Comment:
Say briefly in these comments what is special about each of the 4 BAD / GOOD tests -- e.g. different sources, testing particular sanitizers work ...

Reply:
Added description to each individual test case